airbyte-integrations/bases/base-normalization/normalization/transform_catalog/stream_processor.py

```diff
@@ -138,15 +138,19 @@ def create_from_parent(
 
         The child stream processor will create a separate table to contain the unnested data.
         """
+        if parent.destination_sync_mode.value == DestinationSyncMode.append_dedup.value:
+            # nested streams can't be deduped like their parents (as they may not share the same cursor/primary keys)
+            parent_sync_mode = DestinationSyncMode.append
+        else:
+            parent_sync_mode = parent.destination_sync_mode
         result = StreamProcessor.create(
             stream_name=child_name,
             destination_type=parent.destination_type,
             raw_schema=parent.raw_schema,
             default_schema=parent.default_schema,
             schema=parent.schema,
-            # Nested Streams don't inherit parents sync modes?
-            source_sync_mode=SyncMode.full_refresh,
-            destination_sync_mode=DestinationSyncMode.append,
```

Comment:
Sync modes should be propagated to children tables, otherwise incremental is not going to work properly on nested streams